### ISSUE TITLE
Gradle wrapper distributionUrl is set to ivy.yojoe.local

### ISSUE THREAD

**Issue by franzbecker** [doches/generator-dropwizard-angular-gradle#1]
When I run `yo dropwizard-angular-gradle` I get the following message:
```
Downloading http://ivy.yojoe.local/artifactory/gradle/gradle-2.7-bin.zip

Exception in thread "main" java.net.UnknownHostException: ivy.yojoe.local
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)

```

I noticed that you specified the following distributionUrl for the Gradle wrapper:
```
distributionUrl=http://ivy.yojoe.local/artifactory/gradle/gradle-2.7-bin.zip
```

This looks like an internal mirror which is not reachable by the public and should be:
```
distributionUrl=https\://services.gradle.org/distributions/gradle-2.7-bin.zip
```

**Comment by doches** [comment]
That is indeed a bug! And an egregious one.

**Comment by doches** [comment]
Fixed and released as `0.1.1`; there was also a (latent, but still not-good) link to another of my local mirrors.

Thanks for the catch!

**Comment by franzbecker** [comment]
Works now, thanks! 👍
